### ISSUE TITLE
Recursion warning

### ISSUE THREAD

**Issue by paulray** [nanograv/PINT#854]
I now get this recursion warning when I do anything, even running the simple example fit_NGC6440E.py script:
```
/Users/paulr/src/PINT/src/pint/models/timing_model.py:225: UserWarning: Exception <class 'RecursionError'> maximum recursion depth exceeded while calling a Python object was raised in __getattr__(components)
  type(e), e, name
```
I thought this was fixed in one of the recent PRs?  @aarchiba ?

**Comment by aarchiba** [comment]
This is warning about the bug that will be fixed by #829

**Comment by aarchiba** [comment]
Bug #828
